### ISSUE TITLE
Bad Request: strings must be encoded in UTF-8 (eleirc bot disconnected and not auto connect to the irc server.. )

### ISSUE THREAD

**Issue by shahkacak369** [RITlug/teleirc#332]
Jul 12 18:50:58 myunixbest systemd[1]: Started TeleIRC: Telegram + IRC bridge bot.
Jul 12 18:50:58 myunixbest teleirc[29758]: INFO: 2020/07/12 18:50:58 Current TeleIRC version: v2.0
Jul 12 18:50:58 myunixbest teleirc[29758]: INFO: 2020/07/12 18:50:58 Creating new Telegram bot client...
Jul 12 18:50:58 myunixbest teleirc[29758]: INFO: 2020/07/12 18:50:58 Creating new IRC bot client...
Jul 12 18:50:58 myunixbest teleirc[29758]: INFO: 2020/07/12 18:50:58 Starting up Telegram bot...
Jul 12 18:50:58 myunixbest teleirc[29758]: INFO: 2020/07/12 18:50:58 Starting up IRC bot...
Jul 12 18:50:58 myunixbest teleirc[29758]: INFO: 2020/07/12 18:50:58 Authorized on account teleIRCChatBot
Jul 12 18:55:03 myunixbest teleirc[29758]: ERROR: 2020/07/12 18:55:03 Bad Request: strings must be encoded in UTF-8
Jul 12 18:55:04 myunixbest teleirc[29758]: ERROR: 2020/07/12 18:55:04 Bad Request: strings must be encoded in UTF-8
Jul 12 18:55:04 myunixbest teleirc[29758]: ERROR: 2020/07/12 18:55:04 Bad Request: strings must be encoded in UTF-8

**Comment by Tjzabel** [comment]
@shahkacak369 hello, I see you didn't use our bug template, so I have a few questions:

1. Is there a specific sequence of events that lead up to this issue occuring?
2. Have you done anything to get around this issue in the short-term?
3. What IRC server are you using?
4. Does this issue only occur on IRC, or Telegram?

**Comment by shahkacak369** [comment]
> @shahkacak369 hello, I see you didn't use our bug template, so I have a few questions:
> 
> 1. Is there a specific sequence of events that lead up to this issue occuring?
> 2. Have you done anything to get around this issue in the short-term?
> 3. What IRC server are you using?
> 4. Does this issue only occur on IRC, or Telegram?

hello, do you mean I do not use your bug template?

sequence of events yes as I have explained before that this incident is sudden .. teleirc bot disconnected and did not connect automatically to irc server .. and here I also include the explanation .. along with the journal on the server running teleirc boat .. did you see and read it?

Didn't I already ask you and your team .. what is the cause of all this? and what should I do? am I not waiting for you to give me the steps that need to be done .. for now I can only restart teleirc.service only .. but how many times do I have to restart like this? if this thing happens again and again ..?

The IRC server I use is unrealircd (master.spiderchat.org)

of course this problem occurs on IRC, aTelegram .. aren't irc and telegram now connected?

**Comment by Tjzabel** [comment]
@shahkacak369 We need additional information in order to pinpoint the root cause. This issue should house all information needed for anyone to come to a conclusion and try to work towards a fix. I know we talked over in `#rit-lug-teleirc`, but this issue itself needs additional information that we do not have. 

Yes, this issue is sudden, but we need to figure out why. Additional information can help us:
* Is the bot bridging non-English languages? If so, do specific phrases in this language cause the bot to crash?
  * Of these phrases, are there any that special characters that seem to cause this issue?
* Does this issue only occur from Telegram -> IRC, or IRC -> Telegram traffic?
  * In other words, does this issue only cause the bot to crash from one side of the bridge

----

Showing screenshots, or giving conversation snippets that occur before the bridge dies would be very valuable. I am assuming this issue is in relation to non-ascii characters, but I have nothing to base this off, as we lack information to help us come to an educated conclusion. Thanks :)

**Comment by Tjzabel** [comment]
Some more additional information:

> I started teleirc and tried to send a Cyrillic message from telegram and saw incorrect characters in the regular client.
The server and all users are using CP1251.
